src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -601,6 +601,19 @@ public void testOddAnnotations()
         verifyWarns(checkConfig, fileName, expected);
     }
 
+    @Test
+    public void testAnnotationOddStyles() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(IndentationCheck.class);
+
+        checkConfig.addAttribute("tabWidth", "8");
```

Comment:
please use "addProperty", just merged depreacation of addAttribute

Reply:
Done